src/structures/Message.js

```diff
@@ -426,6 +426,25 @@ class Message extends Base {
     });
   }
 
+  /**
+   * Crossposts the message.
```

Comment:
Clarify what crossposting does:
```suggestion
   * Publishes a message in an announcement channel to all channels following it.
```